netlogo-gui/src/main/app/App.scala

```diff
@@ -847,6 +832,13 @@ class App extends
     showAboutWindow()
   }
 
+  /**
+   * This is called reflectively by the mac app wrapper.
+   */
+  def handleShowPreferences(): Unit = {
+    showPreferencesDialog()
```

Comment:
I want to keep these methods separate so that we don't break the mac-app wrapper when changing them for some other reason.